mac

```diff
@@ -138,7 +138,7 @@ brew "yarn"
 cask "gpg-suite-no-mail"
 
 # Databases
-brew "postgres", restart_service: :changed
+brew "postgresql@14", restart_service: :changed
```

Comment:
Does this actually need to be `postgresql@14` in order to work at all, or would just `postgresql` work?

Reply:
Unfortunately, we have to set an explicit version. If we only use `postgresql`, then Homebrew gives us the following warning.

> Warning: Use postgresql@14 instead of deprecated postgresql

I'd love it if there was a way we could set it to `latest` or similar.